x/qgb/orchestrator/confirm_store.go

```diff
@@ -0,0 +1,134 @@
+package orchestrator
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/celestiaorg/celestia-app/x/qgb/types"
+)
+
+// ConfirmStoreI is a store interface for data commitment confirms and valset confirms.
+type ConfirmStoreI interface {
```

Comment:
[optional][nit]
does the I in `ConfirmStoreI` stand for interface? if we're not planning on naming multiple things ConfirmStore, then I think removing the I improves readability imho

Reply:
will be updated to `QGBStore` in upcoming PRs. Don't want to change it here now not to mess with history and have to handle conflicts